common/lib/xmodule/xmodule/js/spec/capa/display_spec.coffee

```diff
@@ -119,6 +119,13 @@ describe 'Problem', ->
         @problem.render(problem_content_default)
         expect(@problem.$('.problem-progress').html()).toEqual "(2/2 points)"
 
+    describe 'with valid status and and string containing an integer like "0" for detail', ->
+      it 'shows no score possible for the detail', ->
+        @problem.el.data('progress_status', 'foo')
```

Comment:
Nit: you don't have to do this, but if it were me I would create a helper method that takes the progress_status, progress_detail, and expected html after rendering. This would remove code duplication in the various test cases.